src/hooks/useDailyStatistics/utils.ts

```diff
@@ -1,16 +1,25 @@
 import { chain, sumBy } from "lodash";
 import { DailyStatistics, CampaignPolicy } from "../../types";
-import { formatQuantityText } from "../../components/CustomerQuota/utils";
 import { Sentry } from "../../utils/errorTracking";
 
+export type UnitType =
+  | {
+      type: "POSTFIX" | "PREFIX";
+      label: string;
+    }
+  | undefined;
```

Comment:
the default unit was `{ type: "POSTFIX", label: " qty"}`, why are we changing the default value to undefined?